Engine_Revit_UI/Convert/Architecture/FromRevit/Ceilings.cs

```diff
@@ -77,7 +77,10 @@ public static List<oM.Architecture.Elements.Ceiling> CeilingsFromRevit(this Ceil
 
                 newCeiling.UpdateValues(settings, ceiling);
 
-                newCeiling.CustomData.Add("CeilingPattern", ceiling.CeilingPattern(settings, planarSurface));
+                newCeiling.CustomData.Add(BH.Engine.Adapters.Revit.Convert.CeilingPattern, ceiling.CeilingPattern(settings, planarSurface));
+                if (disctionary.Values.Sum(x => x.Count) != 0)
+                    BH.Engine.Reflection.Compute.RecordWarning("Currently ceiling openings are not taken into account when generating ceiling patterns.");
```

Comment:
From my understanding of the code this message should concern not only ceiling patterns, but ceillings as such.